test/test_namedtensor.py

```diff
@@ -741,26 +741,29 @@ def get_minmax_output(t):
             'supports_complete_reduce',
             'supports_multidim_reduce',
             'supports_out_variant',
+            'supports_keepdim',
             'output_lambda',
         ])
 
         tests = [
-            Case('sum', True, True, True, None),
-            Case('prod', True, False, True, None),
-            Case('mean', True, True, True, None),
-            Case('var', True, True, True, None),
-            Case('std', True, True, True, None),
-            Case('std_mean', True, True, False, None),
-            Case('var_mean', True, True, False, None),
-            Case('min', True, False, True, get_minmax_output),
-            Case('max', True, False, True, get_minmax_output),
+            Case('sum', True, True, True, True, None),
+            Case('prod', True, False, True, True, None),
+            Case('mean', True, True, True, True, None),
+            Case('var', True, True, True, True, None),
+            Case('std', True, True, True, True, None),
+            Case('std_mean', True, True, False, True, None),
+            Case('var_mean', True, True, False, True, None),
+            Case('min', True, False, True, True, get_minmax_output),
+            Case('max', True, False, True, True, get_minmax_output),
+            Case('unbind', False, False, False, False, None),
```

Comment:
if all false, what are we testing for `unbind` ?

Reply:
There's a default set of tests that run if everything is false. That default set is {test_simple_reduce(op_name, device)} right now.